docs/whats-new/2020-06.md

```diff
@@ -0,0 +1,139 @@
+---

```

Comment:
The file name should probably be `dotnet-2020-06.md` not just `2020-06.md` to match the other names?

Reply:
I think, not to fix... leave the redundancies out, and in a few months it will all be cleaned up.

Reply:
better for SEO without as well